src/pages/index.jsx

```diff
@@ -17,7 +17,7 @@ import favicon from '../favicon.png';
 import rightArrow from '../right-arrow.svg';
 import github from '../github.svg';
 
-injectGlobal`
+createGlobalStyle`

```

Comment:
You need to create a component and use it in the Index React component:
https://www.styled-components.com/docs/api#createglobalstyle

Reply:
How careless of me! I'll make the correction.